modules/electrophysiology_browser/help/electrophysiology_browser.md

```diff
@@ -0,0 +1,5 @@
+# Electrophysiology Browser
+
+This module allows you to review and download EEG recordings and metadata.  Downloads are organized according the EEG-BIDS standard. 
+
+The selection filter allows you to narrow down candidate sessions. Clicking on any column headers in the data table to sort the table by that column. Individual sessions can be accessed by clicking on the *raw* (for raw datasets) or *all types* (for all data) links of one of the rows under the links column, which will redirect you to the *View Session* page.
```

Comment:
The first 2 sentences violate the help style guide. (The second sentence is also grammatically incorrect.)

Reply:
My bad, I didn't notice the help style guide, I'll read it and review both documents to have them conform to the guide.

Reply:
In particular, I was referring to the part "Assume the user is tech-savvy enough to know about features that are intuitive or common across modern platforms, so you don’t need to repeat in every Help text".. the first example it gives it clicking on column headers to sort, but I think filtering in a selection filter also easily meets that criteria.